examples/query.rs

```diff
@@ -0,0 +1,15 @@
+use rust_fontconfig::{FcFontCache, FcPattern, PatternMatch};
+
+fn main() {
+    let cache = FcFontCache::build();
+    let fonts = cache.query_all(&FcPattern {
+        monospace: PatternMatch::True,
+        ..Default::default()
+    });
+
+    println!("total fonts: {}", fonts.len());
+
+    for font in fonts {
+        println!("{:?}", font);
+    }
+}
```

Comment:
Don't think this needs an extra example as it's just a bugfix.

Reply:
OK, I'll remove it.